<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1102</width>
    <height>776</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Open Sans</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>PTNG Workbench</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/workbench.qrc">
    <normaloff>:/images/goldings_gym_core_00.png</normaloff>:/images/goldings_gym_core_00.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="centralGridLayout">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item row="0" column="0">
     <widget class="QStackedWidget" name="centralStackedWidget">
      <widget class="QWidget" name="imagePage"/>
      <widget class="QWidget" name="tablePage"/>
      <widget class="QWidget" name="xmlPage"/>
      <widget class="QWidget" name="textPage"/>
      <widget class="QWidget" name="codePage"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1102</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNewProject"/>
    <addaction name="actionOpen"/>
    <addaction name="actionOpenRecent"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionShowExplorer"/>
    <addaction name="actionShowOutput"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAboutWorkbench"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <widget class="QMenu" name="menuProject">
    <property name="title">
     <string>Project</string>
    </property>
    <addaction name="actionAddFile"/>
    <addaction name="actionScanFolder"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuProject"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="explorerDockWidget">
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Project Explorer</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="explorerDockWidgetContents">
    <layout class="QGridLayout" name="explorerGridLayout">
     <item row="0" column="0">
      <widget class="QListView" name="listView"/>
     </item>
     <item row="1" column="0">
      <widget class="QListView" name="listViewArtefacts"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="outputDockWidget">
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea|Qt::TopDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Processing Output</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="outputDockWidgetContents">
    <layout class="QFormLayout" name="outputFormLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="clearButton">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Clear</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../res/workbench.qrc">
         <normaloff>:/images/clear_01.png</normaloff>:/images/clear_01.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_2">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Copy all</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../res/workbench.qrc">
         <normaloff>:/images/copy_01.png</normaloff>:/images/copy_01.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QPlainTextEdit" name="outputPlainTextEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionOpenRecent">
   <property name="text">
    <string>Open recent</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionAboutWorkbench">
   <property name="text">
    <string>About Workbench</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionNewProject">
   <property name="icon">
    <iconset resource="../res/workbench.qrc">
     <normaloff>:/images/auto_fix_high_01.png</normaloff>:/images/auto_fix_high_01.png</iconset>
   </property>
   <property name="text">
    <string>New project</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset resource="../res/workbench.qrc">
     <normaloff>:/images/settings_01.png</normaloff>:/images/settings_01.png</iconset>
   </property>
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionShowExplorer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/workbench.qrc">
     <normaloff>:/images/dock_window_04.png</normaloff>:/images/dock_window_04.png</iconset>
   </property>
   <property name="text">
    <string>Show explorer dock window</string>
   </property>
  </action>
  <action name="actionShowOutput">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/workbench.qrc">
     <normaloff>:/images/dock_window_04.png</normaloff>:/images/dock_window_04.png</iconset>
   </property>
   <property name="text">
    <string>Show output dock window</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionAddFile">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/workbench.qrc">
     <normaloff>:/images/folder_open_01.png</normaloff>:/images/folder_open_01.png</iconset>
   </property>
   <property name="text">
    <string>Add file</string>
   </property>
  </action>
  <action name="actionScanFolder">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/workbench.qrc">
     <normaloff>:/images/folder_managed_01.png</normaloff>:/images/folder_managed_01.png</iconset>
   </property>
   <property name="text">
    <string>Scan folder for files</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionNamicsoft">
   <property name="text">
    <string>Namicsoft</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../res/workbench.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>550</x>
     <y>387</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
